### ISSUE TITLE
libtinfo

### ISSUE THREAD

**Issue by jbree** [joeroback/atom-clang#27]
Installing on Fedora 25, after resolving some other installer errors due to required packages you mentioned, I got this error:

`/usr/bin/ld: cannot find -ltinfo`

Ended up resolving it by installing `ncurses-devel` package. Maybe this information will help someone in the future.

**Comment by joeroback** [comment]
yes, thanks, I will update the documentation. Thanks! Appreciate it!
